### ISSUE TITLE
Compile errors: no member named 'size' in 'PixelBuffer<T>'

### ISSUE THREAD

**Issue by pvmm** [openMSX/openMSX#1807]
```
Compiling console/OSDWidget.cc...
In file included from src/console/OSDRectangle.cc:5:
In file included from src/video/GLImage.hh:4:
src/video/GLUtil.hh:227:8: error: no member named 'size' in 'PixelBuffer<T>'
  227 |         other.size = 0;
      |         ~~~~~ ^
1 error generated.
make: *** [build/main.mk:484: derived/x86_64-linux-devel/obj/console/OSDRectangle.cc.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from src/console/OSDImageBasedWidget.cc:5:
In file included from src/video/GLImage.hh:4:
src/video/GLUtil.hh:227:8: error: no member named 'size' in 'PixelBuffer<T>'
  227 |         other.size = 0;
      |         ~~~~~ ^
In file included from src/console/OSDWidget.cc:5:
src/video/GLUtil.hh:227:8: error: no member named 'size' in 'PixelBuffer<T>'
  227 |         other.size = 0;
      |         ~~~~~ ^
In file included from src/console/OSDText.cc:7:
In file included from src/video/GLImage.hh:4:
src/video/GLUtil.hh:227:8: error: no member named 'size' in 'PixelBuffer<T>'
  227 |         other.size = 0;
      |         ~~~~~ ^
1 error generated.
```

**Comment by MBilderbeek** [comment]
It compiles fine here and on GitHub Actions, so it would be helpful if you specify which compiler you're using...

**Comment by pvmm** [comment]
clang version 19.1.0 (Fedora 19.1.0-1.fc41)
Target: x86_64-redhat-linux-gnu

**Comment by m9710797** [comment]
Thanks for reporting!
Fixed in 2fe7d6cde058ea6bce7b7894dcd17b936237d20a.
